src/client/opamCommands.ml

```diff
@@ -3789,6 +3793,8 @@ let admin =
   Term.(ret (const (`Error (true, doc)))),
   Term.info "admin"
```

Comment:
Since you happen to have a diff nearby, we can correct my sins from #4316 here:

```suggestion
  Term.info "admin" ~doc:OpamAdminCommand.admin_command_doc
```